### PR TITLE
Fix examples for no-default-features

### PR DESCRIPTION
**Pull request by Luni-4**
Fixed tests for the `--no-default-features` option

Thanks in advance for your review! :)

### CODE REVIEW

**Comment by est31** [comment]
@Luni-4 thanks for the PR. Do you know how to enable github ations? They don't seem to run for this PR.

**Comment by Luni-4** [comment]
> @Luni-4 thanks for the PR. Do you know how to enable github ations? They don't seem to run for this PR.

Oh, no! They don't work on master too. I guess they are blocked by the `travis` bot. Could you try to disable it, please?

File: examples/player.rs [hunk 1]
```diff
@@ -8,12 +8,18 @@ extern crate alto;
 extern crate lewton;
 extern crate byteorder;
 
+#[cfg(feature = "ogg")]
```
Comment (est31) [inline examples/player.rs]: Can you instead add a `#![cfg(feature ... )]` to the top of the file?

**Comment by est31** [comment]
@Luni-4 it's disabled. Still not working.

**Comment by Luni-4** [comment]
> @Luni-4 it's disabled. Still not working.

Have you tried to force push `master`?

**Comment by est31** [comment]
Force pushed, still not working.

**Pull request closed without merging**